src/test/kotlin/uk/gov/dluhc/printapi/config/LocalStackContainerConfiguration.kt

```diff
@@ -117,4 +132,86 @@ class LocalStackContainerConfiguration {
             ).toASCIIString()
         }
     }
+
+    @Primary
+    @Bean
+    fun testDynamoDbClient(
+        testAwsCredentialsProvider: AwsCredentialsProvider,
+        dbConfiguration: DynamoDbConfiguration
+    ): DynamoDbClient {
+        val dynamoDbClient = DynamoDbClient.builder()
+            .credentialsProvider(testAwsCredentialsProvider)
+            .endpointOverride(localStackContainer.getEndpointOverride())
+            .build()
+
+        createPrintDetailsTable(dynamoDbClient, dbConfiguration.printDetailsTableName)
+        return dynamoDbClient
+    }
+
+    private fun createPrintDetailsTable(dynamoDbClient: DynamoDbClient, tableName: String) {
+        if (dynamoDbClient.listTables().tableNames().contains(tableName)) {
+            dynamoDbClient.deleteTable { it.tableName(tableName) }
```

Comment:
Since we're reusing containers, I added this to make sure we have the latest version of the table.

Reply:
Guess during development this is only ran once and helps development 👍 (I guess with MySQL we don't do anything similar - although Liquibase will complain if an applied changeset is modified)